template_config.yml

```diff
@@ -5,7 +5,7 @@
 
 additional_repos:
 - bindings: true
-  branch: 0.16.0
+  branch: main
```

Comment:
While we use pulp* `main`, will we need to occasionally run the `plugin-template` script and requirements `make` cmd to ensure we bring in the latest template changes and requirement updates?

Reply:
It should always bring in the main branch. I just fixed something in pulp ansible that was breaking ci and I didn't have to update anything on the pr other than rerunning the tests